src/rez/rezconfig.py

```diff
@@ -38,7 +38,8 @@
 
 Paths should use the path separator appropriate for the operating system
 (based on Python's os.path.sep).  So for Linux paths, / should be used. On
-Windows \ (unescaped) should be used.
+Windows \\ (unescaped! A change in python 3.12 does not allow single backslashes
```

Comment:
We could add `r` do the string like we usually do with regexes. THis will allow us to have a single `\`.